tests/test_routes_decisions_agent.py

```
@@ -702,3 +702,32 @@ async def test_agent_multi_select_other_plus_option_accepted(client):
+
+@pytest.mark.asyncio
+async def test_agent_cannot_answer_via_human_path(client):
+    """An agent with a bearer JWT must NEVER be able to answer its own question
+    via the human path POST /api/decisions/{id}/answer. That route is not in
+    _AGENT_DECISIONS_ROUTES, so the auth middleware does not pass the Bearer
+    through, current_user_or_device falls through to require_device (which
+    rejects a non-device token), and the request 401s. The answer is always
+    attributed to the real user identity (session/device), never the agent."""
+    app = client._transport.app
+    pid = await _new_project(client)
+    cid, token = await _mint_agent(app, pid, ("decisions_write",))
```

Ruff (0.16.1)

[warning] Unpacked variable `cid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)